CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix VerifySchema invalid header error when VerifySchema called without data entry.
```

Comment:
The changelog message should be changed to reflect the new changes, something about using the app's authtoken instead of the user's.